plugins/modules/postgresql_privs.py

```diff
@@ -1093,14 +1096,28 @@ def main():
         else:
             privs = None
         # objs:
-        if p.type == 'table' and p.objs == 'ALL_IN_SCHEMA':
-            objs = conn.get_all_tables_in_schema(p.schema)
-        elif p.type == 'sequence' and p.objs == 'ALL_IN_SCHEMA':
-            objs = conn.get_all_sequences_in_schema(p.schema)
-        elif p.type == 'function' and p.objs == 'ALL_IN_SCHEMA':
-            objs = conn.get_all_functions_in_schema(p.schema)
-        elif p.type == 'procedure' and p.objs == 'ALL_IN_SCHEMA':
-            objs = conn.get_all_procedures_in_schema(p.schema)
+        if p.objs == 'ALL_IN_SCHEMA':
```

Comment:
```suggestion
        orig_objs = None
        if p.objs == 'ALL_IN_SCHEMA':
```
i thin it'll fix the integration test error `UnboundLocalError: local variable 'orig_objs' referenced before assignment`